tests/Browser/Pages/Backend/Categories/AdminEditCategoriesTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace Tests\Browser\Pages\Backend\Categories;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use App\Model\User;
+use App\Model\Category;
+use Tests\Browser\Pages\Backend\Users\BaseTestUser;
+
+class AdminEditCategoriesTest extends BaseTestUser
+{
+    use DatabaseMigrations;
+
+    /**
+    * Override function setUp()
+    *
+    * @return void
+    */
+    public function setUp()
+    {
+        parent::setUp();
+        $this->makeAdminUserToLogin();
```

Comment:
ở đây em : $this->user = $this->makeAdminUserToLogin()

Reply:
em có làm rồi anh